addons/web/static/src/core/debug/profiling/profiling_item.xml

```diff
@@ -18,6 +18,14 @@
                                 t-att-checked="profiling.isCollectorEnabled('sql')"/>
                             <label class="form-check-label" for="profile_sql">Record sql</label>
                         </span>
+                        <div t-if="profiling.isCollectorEnabled('sql')" class="input-group input-group-sm mt-2" t-on-click.stop.prevent="">
+                            <div class="input-group-text">Constant Time</div>
+                            <select class="profile_param form-select" t-on-change="(ev) => this.changeParam('constant_time', ev)">
+                                <t t-set="constant_time" t-value="profiling.state.params.constant_time"/>
```

Comment:
I'm not sure it is the right place for the constant time selection. This menu is to enable or disable a recorder, profile_params are for recorders.

Constant_time is an option on display, so it could be selectionable from the backend, near the link to access the result.

Actually it is part of the parameters for speedscope, so it should in the same place where we select what will be included in the speedscope render. Could be add in the context of the render making the field context dependant, or move the computed field on a Transient model (not sure what would be the best).